frontend/admin/src/pages/purchase/PurchaseDetail.tsx

```diff
@@ -164,6 +175,10 @@ const PurchaseDetailInternal = (props: any) => {
                     {purchase.status === 'Neu' &&
                     <Button className="mr-3 mb-1"
                             onClick={() => publishPurchaseSearchHelper()}>Einkauf freigeben (Helfer suchen)</Button>}
+                    {(purchase.status === 'Neu' || purchase.status === 'Veröffentlicht' || purchase.status === 'Helfer gefunden') && <>
```

Comment:
Or use `purchase.assignedVolunteer && <>...</>` to match the condition in the backend